cmd/cspi-mgmt/app/handler.go

```diff
@@ -92,7 +99,14 @@ func (c *CStorPoolInstanceController) reconcile(key string) error {
 	}
 	common.SyncResources.Mux.Unlock()
 
-	return c.updateStatus(cspi)
+	err = c.updateStatus(cspi)
+	if err != nil {
+		c.recorder.Event(cspi,
```

Comment:
Here we are logging the event two times if CR update fails in `upadateStatus`.

Reply:
removed